src/System.Text.Utf8/src/System/Text/Utf8/Utf8EncodedCodePoint.cs

```diff
@@ -21,7 +21,7 @@ public unsafe Utf8EncodedCodePoint(char character) : this()
 
             fixed (byte* encodedData = &_byte0)
             {
-                ByteSpan buffer = new ByteSpan(encodedData, 4);
+                Span<byte> buffer = new Span<byte>(encodedData, 4);
```

Comment:
In many cases I agree with you, but when you new up stuff, it's pretty clear what the type is.